common/streams/Cargo.toml

```diff
@@ -13,23 +13,24 @@ test = false
 [dependencies] # In alphabetical order
 # Workspace dependencies
 common-arrow = {path = "../arrow"}
-common-base = {path = "../base" }
+common-base = {path = "../base"}
+common-dal = {path = "../dal"}
 common-datablocks = {path = "../datablocks"}
 common-datavalues = {path = "../datavalues"}
 common-exception = {path = "../exception"}
+common-functions = {path = "../functions"}
```

Comment:
It's Clion's auto-reorder?  I use vscode fail to find this format commands.

Reply:
It's in vscode too, the command name is Format Cell:)
I found the order of almost all config in package section,  is name->version->description..., should I recover this?

Reply:
No, it's better to have alpha orders.